src/lamp_py/ingestion_tm/jobs/parition_table.py

```diff
@@ -182,8 +160,122 @@ def run_export(self, tm_db: MSSQLManager) -> None:
                     )
                     upload_file(local_pq, s3_export_path)
 
-            self.update_version_file()
-            logger.log_complete()
+                logger.log_complete()
 
-        except Exception as exception:
-            logger.log_failure(exception)
+            except Exception as exception:
+                logger.log_failure(exception)
+
+        self.update_version_file()
```

Comment:
`update_version_file` should be in the `try` block and before `log_complete` or have it's own logging/error catching

Reply:
moved it into the for loop.